conf/defaults.ini

```diff
@@ -366,6 +366,17 @@ client_secret = some_secret
 scopes = user:email
 allowed_organizations =
 
+#################################### Azure AD OAuth #######################
+[auth.azuread]
```

Comment:
This should be in `sample.ini` as well I guess.

Reply:
Sorry, I haven't seen that file before. I added the section there as well.